securedrop_client/app.py

```diff
@@ -221,23 +234,25 @@ def start_app(args, qt_args) -> NoReturn:  # type: ignore [no-untyped-def]
     app_state = state.State(database)
     gui = Window(app_state)
 
-    controller = Controller(
-        "http://localhost:8081/",
-        gui,
-        session_maker,
-        args.sdc_home,
-        app_state,
-        not args.no_proxy,
-        not args.no_qubes,
-    )
-    controller.setup()
-
-    configure_signal_handlers(app)
-    timer = QTimer()
-    timer.start(500)
-    timer.timeout.connect(lambda: None)
-
-    sys.exit(app.exec_())
+    with thread() as export_thread:
```

Comment:
note (for me mostly): we create this once when the application starts and we create a Controller... the follow-up PRs handle managing the sync thread and queue thread